lbr_fri_ros2_stack/repos-fri-2.6.yaml

```diff
@@ -0,0 +1,13 @@
+repositories:
+  fri:
+    type: git
+    url: https://github.com/lbr-stack/fri
+    version: fri-2.6
```

Comment:
refer to this issue here https://github.com/lbr-stack/fri/issues/33#issuecomment-2687372188